rotkehlchen/tasks/manager.py

```diff
@@ -279,6 +282,19 @@ def _maybe_schedule_exchange_history_query(self) -> None:
         )
         self.last_exchange_query_ts[exchange.location_id()] = now
 
+    def _maybe_query_missing_prices(self) -> None:
+        query = HistoryEventFilterQuery.make()
+        task_name = 'Periodically query history events prices'
+        log.debug(f'Scheduling task to {task_name}')
+        self.greenlet_manager.spawn_and_track(
+            after_seconds=None,
+            task_name=task_name,
+            exception_is_error=True,
```

Comment:
I would advise here, just like with other tasks to limit it a bit. It can become a really long runnig tasks if there is say 1000 entries without a price.

Also there can be entries that do not have a price and there is no way to find one with the current oracles. So the scheduler will re-schedule, and re-schedule, and re-schedule, and re-schedule, and re-schedule .... you get the drift.

For example we face this problem where querying coingecko for icons. Which is why we keep the ids of assets for which we could not find an icon in memory so they are not scheduled again: https://github.com/rotki/rotki/blob/32a9010576023cf69c6202731480d9b00136e966/rotkehlchen/icons.py#L41
https://github.com/rotki/rotki/blob/32a9010576023cf69c6202731480d9b00136e966/rotkehlchen/icons.py#L88
https://github.com/rotki/rotki/blob/32a9010576023cf69c6202731480d9b00136e966/rotkehlchen/icons.py#L164-L166